tests/ha/ha_cluster_init.pm

```diff
@@ -92,7 +93,11 @@ sub run {
     # This may be necessary if your cluster nodes reboot so fast that the
     # other nodes are still waiting in the fence acknowledgement phase.
     # This is an occasional issue with virtual machines.
-    file_content_replace("$sbd_cfg", "SBD_DELAY_START=.*" => "SBD_DELAY_START=yes");
+    # In case of newer SLES (15SP4+), it performs better leaving it on default values.
+    if (is_sle('<15-SP4')) {
+        record_info("SBD_DELAY_START set to YES");
+        file_content_replace("$sbd_cfg", "SBD_DELAY_START=.*" => "SBD_DELAY_START=yes");
+    }
```

Comment:
I think you can just replace this whole block with :
setup_sbd_delay() if (is_sle('<15-SP4')); 

And set openQA parameter HA_SBD_START_DELAY='yes'

This will set both SBD delay pacmeaker parameter and sbd service timeout.